dags.yaml

```diff
@@ -57,7 +60,7 @@ bqetl_vrbrowser:
     retry_delay: 5m
 
 bqetl_core:
-  schedule_interval: 0 2 * * *
+  schedule_interval: 15 2 * * *
```

Comment:
Requires changes in https://github.com/mozilla/telemetry-airflow/blob/fd54979f705b7d402b910407b79e099162b36f91/dags/incline_dash.py#L37